### DISCUSSION TITLE
[@types/jsforce] types is missing `requestPost` in the `Connection` class

### DISCUSSION THREAD

**Discussion by kylejw2** [DefinitelyTyped/DefinitelyTyped#57653] [Issues with a @types package]
JSForce types is missing `requestPost` in the `Connection` class. This method is cited across websites as the main method for a composite API, but it's nowhere to be found in the current types for the `Connection` class. I'd be happy to contribute to speed up getting access to an updated version of the types.

[a comment by an automated account is not shown]

**Comment by dolanmiu** [comment]
Sure yeah, if you can make a PR, that would be great. I will review and approve

**Comment by martin-badin** [comment]
PR is ready #57660

**Reply by dolanmiu** [reply]
nice, saw it, approved
